### ISSUE TITLE
FoulDecrypt stacks on iproxy starting

### ISSUE THREAD

**Issue by Numenorean** [ChiChou/vscode-frida#24]
Windows 10, iproxy exists in the path. Commands like which and zip are available

**Comment by ChiChou** [comment]
Any traces in OUTPUT > Frida Extension?

**Comment by Numenorean** [comment]
> Any traces in OUTPUT > Frida Extension?

`[iproxy ssh] recv failed: connection reset`
however there is a process iproxy in process manager

**Comment by ChiChou** [comment]
Thank you for reporting

**Comment by Numenorean** [comment]
still doesn't work
```
[iproxy ssh] recv failed: connection reset
[iproxy ssh] send failed: No error
```

**Comment by ChiChou** [comment]
increased the timeout
https://github.com/ChiChou/vscode-frida/commit/717fa6f989e911165354042500cf5575d927afe8#diff-3661c7de958e281100120f53f7416c2982f5081c251ae6a92d4bb7bc63a16ba6L33

**Comment by Numenorean** [comment]
now just `[iproxy ssh] recv failed: connection reset`

**Comment by ChiChou** [comment]
> now just `[iproxy ssh] recv failed: connection reset`

That's strange...

**Comment by Chinksy** [comment]
I'm having the same problem. First it shows the first 2 lines and after a minute of so it show the other 2:
```
[iproxy ssh] recv failed: connection reset
[iproxy ssh] send failed: No error
[iproxy ssh] recv failed: connection reset
[iproxy ssh] recv failed: Unknown error
```

**Comment by Numenorean** [comment]
solved, it was an issues with my ssh configuration. Added a private key to the ssh-agent with command ssh-add. I guess it stacked because all the time ssh asked for the password to read private key

**Comment by Chinksy** [comment]
@Numenorean Woah that actually worked! Thank you!

For anyone on Windows these are the extra steps I noticed I had to do to complete the smooth SSH login:
1. Add the id_rsa.pub RSA key to the authorized_keys on the iPhone (do this via ssh, I suggest just plain WinSCP)
2. Start the "OpenSSH Authentication Agent" service on your PC [[SO answer]](https://stackoverflow.com/questions/18683092/how-to-run-ssh-add-on-windows/40720527#40720527)
3. In cmd run `ssh-agent` (no output)
4. In cmd run `ssh-add C:\path\to\your\key\id_rsa` (output should be "Identity added")

@ChiChou Maybe the "connection reset" error should be modified to hint in this direction?
